Source/Scene/SkyAtmosphere.js

```diff
@@ -81,6 +81,42 @@ define([
         this._spSkyFromSpace = undefined;
         this._spSkyFromAtmosphere = undefined;
 
+        this._spSkyFromSpaceColorCorrect = undefined;
+        this._spSkyFromAtmosphereColorCorrect = undefined;
+
+        /**
+         * Use hue/saturation/brightness shifts to postprocess the sky/atmosphere color.
+         * @type (Boolean)
+         * @default false
+         */
+        this.colorCorrect = false;
```

Comment:
Mainly this is just from a usability standpoint, the user doesn't need to know to turn on `colorCorrect` first, it just works when changing hue, saturation, or brightness.